lib/linguist/languages.yml

```diff
@@ -5278,8 +5278,20 @@ Text:
   wrap: true
   aliases:
   - fundamental
+  - change-log
+  - changelog
+  - outline
   extensions:
   - ".txt"
+  - ".1"
```

Comment:
What about `.0`?

Reply:
We don't associate that extension with Roff, because there's never been a `man0` section…